packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -26,7 +26,7 @@ Obsoletes: rubygem-hammer_cli_gutterball
 
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.13.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.14.0
```

Comment:
I see the gemspec fully dropped the minimum version requirement. Not sure that's wise and I'd prefer to keep it in the gemspec.